recipes-support/provisioning/phytec-board-config.bb

```diff
@@ -0,0 +1,25 @@
+SUMMARY = "Device registration to ESEC IoT Device Manager plattform"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = " \
+    file://phytec-board-config.sh \
+"
+
+S = "${WORKDIR}"
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 0500 ${S}/phytec-board-config.sh ${D}${bindir}/phytec-board-config
+}
+
+FILES_${PN} = " \
+    ${bindir}/phytec-board-config \
+"
+# Runtime packages used in 'esec-device-onboarding.sh'
+RDEPENDS_${PN} = " \
+    bash \
```

Comment:
This dependency on BASH can be removed now.

Reply:
is removed